doc/strftime_formatting.rdoc

```diff
@@ -461,3 +461,67 @@ The integer width specifier gives a minimum width for the returned string:
   Time.new(2002, 12).strftime('%B')   # => "December" # No width specifier.
   Time.new(2002, 12).strftime('%10B') # => "  December"
   Time.new(2002, 12).strftime('%3B')  # => "December" # Ignored if too small.
+
+== Specialized Format Strings
```

Comment:
Would it make sense to move the "ISO 8601 Format Specifications" section under this? ISO 8601 is a specialized format string according to a specific standard.

Reply:
Done.